pages/common/restic.md

```diff
@@ -0,0 +1,23 @@
+# restic
+
+> Fast, secure, efficient backup program.
+
+- Initialize a backup repository in the specified local directory:
+
+`restic init -r {{/path/to/repository/}}`
+
+- Backup folder "my_folder" to the repository:
+
+`restic -r {{/path/to/repository/}} backup {{/path/to/my_folder}}`
+
+- Show backup snapshots currently stored in the repository:
+
+`restic -r {{/path/to/repository/}} snapshots`
+
+- Restore a specific backup snapshot to a target directory:
+
+`restic -r {{/path/to/repository/}} restore {{snapshot_id}} {{/path/to/target}}`
+
+- Clean up repository and keep only the most recent snapshot of each unique backup (same host and directory):
```

Comment:
Do we need to mention "same host and directory" ? I thought that was implied.

Reply:
There are no default interval when snapshots are taken, they are taken whenever you run the backup command. For regular automatic backups you'll need to use cron or something.

You're right, its probably redundant – removed!